server/node/src/main/java/org/kaaproject/kaa/server/control/service/DefaultControlService.java

```diff
@@ -254,28 +232,38 @@
     @Autowired
     @Qualifier("rootCredentialsServiceLocator")
     private CredentialsServiceLocator credentialsServiceLocator;
-    
+
     @Autowired
     private CredentialsServiceRegistry credentialsServiceRegistry;
-    
+
     @Autowired
     private EndpointRegistrationService endpointRegistrationService;
 
-    /** The neighbor connections size. */
+    /**
+     * The neighbor connections size.
+     */
     @Value("#{properties[max_number_neighbor_connections]}")
     private int neighborConnectionsSize = DEFAULT_NEIGHBOR_CONNECTIONS_SIZE;
 
-    /** The user hash partitions. */
+    /**
+     * The user hash partitions.
+     */
     @Value("#{properties[user_hash_partitions]}")
     private int userHashPartitions = DEFAULT_USER_HASH_PARTITIONS_SIZE;
 
-    /** The neighbors. */
+    /**
+     * The neighbors.
+     */
     private volatile Neighbors<NeighborTemplate<OperationsServiceMsg>, OperationsServiceMsg> neighbors;
 
-    /** The resolver. */
+    /**
+     * The resolver.
+     */
     private volatile OperationsServerResolver resolver;
 
-    /** The zk lock. */
+    /**
+     * The zk lock.
```

Comment:
All these comments are ultimately useless and don't make the code better but rather harm it. (For more reasoning see https://bradt.ca/blog/useless-code-comments/)